### ISSUE TITLE
Colour Loss

### ISSUE THREAD

**Issue by Lendud** [ionar2/salhack#197]
**Describe the bug**
After installing the most up to date version of Salhack (2.03), colours for things such as; VoidESP and HoleESP have vanished.

**To Reproduce**
1. Go to 'GUI'
2. Click on 'Holesp'
3. See lack of colour

**Expected behavior**
Green and Blue depending on hole.

**Additional context**
I even tried fresh installing salhack and nothing changed.

**Comment by Tyler2b2t** [comment]
I also have this issue, changing the RGB settings doesn't effect anything

**Comment by ionar2** [comment]
Should be fixed. 5e5ddcb566da90f3ebc69220f913a76f95e0fb8f

**Comment by Lendud** [comment]
^^ not fixed

**Comment by Tyler2b2t** [comment]
yeah baritone is causing the colors to turn grey, but even without baritone it lags quite a bit.
